README.md

```diff
@@ -13,63 +13,19 @@ for implementation details.
 In order to use this server, you should have:
 
 - This project checked out. You may wish to create a GitHub Fork of the project and check that out instead so that you can customize the project and save the results to GitHub.
-
-### and either
- - Oracle Java (JDK) installed: Minimum JDK8 or newer.
- - Apache Maven build tool (newest version)
-
-### or
- - Docker, as the entire project can be built using multistage docker (with both JDK and maven wrapped in docker).
-
-## Running locally
-
-The easiest way to run this server entirely depends on your environment requirements. At least, the following 4 ways are supported:
-
-### Using jetty
-```bash
-SPRING_PROFILES_ACTIVE="local" mvn -Djetty.port=8888 jetty:run
-```
```

Comment:
I think we should leave this so folks can use it as a regular Java process connected to an existing DB.

Reply:
It seems it would be more beneficial for us to support a single/consistent development environment across our repos.

Otherwise, we'll have to always update these instructions in multiple places, could run into issues in one config and not the other, and etc.

Reply:
Discussed offline with @leite08 - will keep the jetty/mvn instructions in another section & remove the docker one